### DISCUSSION TITLE
How to fix this issue, when opening tlauncher with modpack it crashes

### DISCUSSION THREAD

**Discussion by Jhovanyy090** [FabricMC/fabric-api#4048] [FAQ]
![Uploading IMG_20240822_144452.jpg…]()

**Comment by Linguardium** [comment]
<https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies#2-user-safety> First bullet
<https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies#3-intellectual-property-authenticity-and-private-information> First bullet here too
